### ISSUE TITLE
Hama not showing up on latest Plex Media Server on macOS (Follow up)

### ISSUE THREAD

**Issue by Caf** [ZeroQI/Hama.bundle#361]
My 1st thread was [here](https://github.com/ZeroQI/Hama.bundle/issues/353).
**Context:** long story short, brand new Mac Mini 2018 had faulty SSD, that finally crashed and burned... ;) Took time to get a new one (they have to change the whole motherboard...) and I decided to reinstall everything from scratch. As of now, all my download stack is up & running fine (Sonarr, Radarr, Jackett, NZBGet, etc. and of course Plex). Plex is the latest version (server 1.18.1.1973)

SO, time to reinstall Hama + Absolute Series Scanner. AND, **all is working FINE. Yeah, no issue here**, I just wanted to say that something was obviously wrong with that SSD and point out just one thing ➜

**The Plex Logs:**
**They moved**, at least on macOS, you should update that in the docs and when telling people where to look. **It's in (userdir)/library/Logs/Plex Media Server/PMS Plugin Logs** now. You'll find nothing in Application Support/Plex Media Server. The menus in PMS itself changed too (no more server category, "Agents" is inside Settings>Settings now)

TL;DR : Thanks again for your work and your help. I just wanted to thank you. Thought it would change a bit compared to the usual threads. :p Have a great day!

**Comment by ZeroQI** [comment]
Thanks for letting me know and inaugurating the template system
